test/testdata/sandbox_config_seccomp.json

```diff
@@ -47,7 +47,6 @@
 	},
 	"annotations": {
 		"owner": "hmeng"
-		%VALUE%
```

Comment:
oh tests are failing because this has been removed, this must be kept in order to test the various seccomp stuff (it's just a placeholder to be replaced with sed and a default seccomp annotation).
@rhatdan could you put this back?